lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -229,18 +229,18 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
         #     # of sql queries to default,
         #     # of mongo queries,
         # )
-        ('no_overrides', 1, True, False): (22, 6),
-        ('no_overrides', 2, True, False): (22, 6),
-        ('no_overrides', 3, True, False): (22, 6),
-        ('ccx', 1, True, False): (22, 6),
-        ('ccx', 2, True, False): (22, 6),
-        ('ccx', 3, True, False): (22, 6),
-        ('no_overrides', 1, False, False): (22, 6),
-        ('no_overrides', 2, False, False): (22, 6),
-        ('no_overrides', 3, False, False): (22, 6),
-        ('ccx', 1, False, False): (22, 6),
-        ('ccx', 2, False, False): (22, 6),
-        ('ccx', 3, False, False): (22, 6),
+        ('no_overrides', 1, True, False): (25, 6),
```

Comment:
This should have increased by only 1, correct?  Since it hasn't, the profile query is probably not being cached in the request - and may indicate a scalability issue (querying for each subsection in the course).